### PR TITLE
Reproducible build

### PR DESCRIPTION
**Pull request by aalmiray**
Updates parent pom to make build reproducible.
As a side effect the code-formatter was also updated hence why so many source files have also been updated.

### CODE REVIEW

**Comment by aalmiray** [comment]
Supersedes #177

**Comment by hboutemy** [comment]
@aalmiray any blocker before merging and releasing?

**Comment by aalmiray** [comment]
@hboutemy waiting for a review from @gunnarmorling as the changes posted are more extensive that I thought

File: core/src/main/java/org/moditect/Moditect.java
Comment (gunnarmorling) [inline core/src/main/java/org/moditect/Moditect.java:45]: Is this actually more readable? Not that I'm interested in having a formatting style discussion, but could we configure it so it doesn't join wrapped lines? At least there's such an option in the Eclipse formatter. If it's not doable here for some reason, let's go with the proposed change.

**Review by gunnarmorling — APPROVED** [review]
LGTM. One question inline, but it's not a blocker. Feel free to merge one way or the other. Thanks!

File: core/src/main/java/org/moditect/Moditect.java
Comment (aalmiray) [reply core/src/main/java/org/moditect/Moditect.java:45]: I took the formatting configuration from the moditect-org-parent project

Comment (gunnarmorling) [reply core/src/main/java/org/moditect/Moditect.java:45]: Ok, can we then perhaps add the "don't join wrapped lines" config there?

**Comment by hboutemy** [comment]
@aalmiray time for merge and release?

File: core/src/main/java/org/moditect/Moditect.java
Comment (aalmiray) [reply core/src/main/java/org/moditect/Moditect.java:45]: As far as I can tell it's already there

```
<setting id="org.eclipse.jdt.core.formatter.join_wrapped_lines" value="false"/>
```

**Comment by hboutemy** [comment]
FYI: i just checked RB for this release and all went well!!!
https://github.com/jvm-repo-rebuild/reproducible-central/tree/master/content/org/moditect/moditect

the plugin itself is reproducible
now let's check that plugins users now get a reproducible output from this release also = the most expected benefit :)

**Pull request merged**
